src/sql-parser/src/ast/defs/statement.rs

```diff
@@ -904,6 +904,7 @@ impl_display!(DiscardTarget);
 pub struct DropDatabaseStatement {
     pub name: Ident,
     pub if_exists: bool,
+    pub cascade: bool,
```

Comment:
If `CASCADE` is the default, I think specifying the `bool` as `RESTRICT` makes a little more sense. This is opposite what e.g. `DROP INDEX` does, but I believe that's because `RESTRICT` is the default, so these conditional variables should be true only when a non-default behavior's applied.